packages/core/src/odata-common/request-builder/batch/batch-response-parser.ts

```diff
@@ -126,7 +126,9 @@ export function splitResponse(response: string, boundary: string): string[] {
   const newLineSymbol = detectNewLineSymbol(response);
   const parts = response.split(`--${boundary}`).map(part => {
     const trimmedPart = part.trim();
-    return trimmedPart.includes('204 No Content')
+    return trimmedPart.includes('204 No Content') ||
+      (trimmedPart.includes('200 OK') &&
```

Comment:
What happens in the case of `200 OK` but the there is content, is there in this case perhaps a line to much. See comment on test.

Reply:
Good point. Unfortunately, in the current SDK, we assume update and delete response contains nothing.
- for a normal delete request, the response type of `execute` is `void`
- for a normal update request, the response is ignored and we build remote state as return value.

I personally would like to touch the API to reflect the real response.

Reply:
@FrankEssenberger : I didn't see a comment on test. 
From my understanding, if the status code is `200 OK` and response has some payload, then `Content-Length` should not be 0. The condition I added checks both:
`(trimmedPart.includes('200 OK') && trimmedPart.includes('Content-Length: 0'))` 

Therefore, when response has payload, we won't add another line. We have another test for this scenario (Line 279).